src/Microsoft.ML.AutoML/Utils/MLNetUtils/Contracts.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.AutoML
+{
+    internal static class Contracts
```

Comment:
??? Why not use the existing `Contracts` from ML.NET and not duplicate the whole thing?

Reply:
yeah, ugh. done